stock_available/models/res_config.py

```diff
@@ -31,3 +42,27 @@ class StockConfig(models.TransientModel):
              "This installs the module stock_available_mrp.\n"
              "If the module mrp is not installed, this will install it "
              "too")
+
+    stock_available_mrp_based_on = fields.Selection(
+        _get_stock_available_mrp_based_on,
+        string='based on',
+        help="Choose the field of the product which will be used to compute "
+             "potential.\nIf empty, Quantity On Hand is used.\n"
+             "Only the quantity fields have meaning for computing stock",
+    )
+
+    @api.model
+    def get_default_stock_available_mrp_based_on(self, fields):
```

Comment:
coveralls says this methods are never called and I don't see it used anywhere either.
Should you remove it ?